### PR TITLE
[DEV-207951] react native  render widget embedded on page

### PR DESCRIPTION
**Pull request by marcosrivereto**

### CODE REVIEW

**Review by pedroabdenor — APPROVED** [review]
👍

**Review by lukevance — CHANGES_REQUESTED** [review]
This may not be important at the moment, so tell me what the effort level is here, but ideally the lightbox will launch when the user selects the bank as it does in our other example apps. If that's a major change we can consider leaving it like this for now, but the current implementation has this iOS component at the bottom of the screen that hides the "Pay" button after I select the bank. Can we fix that?

https://github.com/TrustlyInc/trustly-react-native-example/assets/13320316/934ba22d-fdb2-4d04-ba7b-139a83187e41

File: src/trustly.tsx [hunk 1]
```diff
@@ -4,18 +4,48 @@ export const widget = (data = {}) => `<!DOCTYPE html>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Trustly</title>
-  <script src="https://sandbox.trustly.one/start/scripts/pwmb.js?accessId=<YOUR_ACCESS_ID>" type="text/javascript"></script>
+  <script src="https://sandbox.trustly.one/start/scripts/pwmb.js?accessId=YOUR_ACCESS_ID>" type="text/javascript"></script>
```
[the remark this answers is not in the document]
Comment (marcosrivereto) [reply src/trustly.tsx:7]: Done

File: src/trustly.tsx [hunk 2]
```diff
@@ -4,18 +4,48 @@ export const widget = (data = {}) => `<!DOCTYPE html>
...
+</body>
+</html>`;
+
+export const lightbox = (data = {}) => `<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Trustly</title>
+  <script src="https://sandbox.trustly.one/start/scripts/pwmb.js?accessId=YOUR_ACCESS_ID>" type="text/javascript"></script>
```
[the remark this answers is not in the document]
Comment (marcosrivereto) [reply src/trustly.tsx:35]: Done

File: src/App.tsx [hunk 3]
```diff
@@ -18,12 +18,16 @@ import { MaskedTextInput} from "react-native-mask-text";
 export default class App extends Component {
   trustlyWebView = null;
 
+  ACCESS_ID = "<YOUR_ACCESS_ID>";
```
Comment (lukevance) [inline src/App.tsx:21]: 🙌

File: src/trustly.tsx [hunk 4]
```diff
@@ -1,10 +1,10 @@
-export const widget = (data = {}) => `<!DOCTYPE html>
+export const widget = (accessId: String, data = {}) => `<!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Trustly</title>
-  <script src="https://sandbox.trustly.one/start/scripts/pwmb.js?accessId=YOUR_ACCESS_ID>" type="text/javascript"></script>
+  <script src="https://sandbox.trustly.one/start/scripts/trustly.js?accessId=${accessId}" type="text/javascript"></script>
```
Comment (lukevance) [inline src/trustly.tsx:7]: 🕶️

File: src/trustly.tsx [hunk 5]
```diff
@@ -26,13 +26,13 @@ export const widget = (data = {}) => `<!DOCTYPE html>
 </body>
 </html>`;
 
-export const lightbox = (data = {}) => `<!DOCTYPE html>
+export const lightbox = (accessId: String, data = {}) => `<!DOCTYPE html>
```
Comment (lukevance) [inline src/trustly.tsx:29]: 🦸

**Review by lukevance — APPROVED** [review]
Looks good and everything is working as expected now! 🥇

**Pull request merged**